src/App.js

```diff
@@ -1,21 +1,10 @@
-import React from 'react';
+import { EntryList } from "./EntryList";
```

Comment:
You have exported EntryList as `default` and in this case you need to import it as default 

like this

import EntryList from "./EntryList" 

note I used `EntryList` here as name but you can use any name you like 

you can do 

`import Foo from "./EntryList"`
or 
`import Abcdef from "./EntryList"`

Reply:
in our previous project :
import { Flowers } from "./Flowers";

it worked for us 
also we had in the end:
export default App;

Reply:
![image](https://user-images.githubusercontent.com/60723932/173186446-9acf28e8-415f-4146-8c23-df257aa15d92.png)

![image](https://user-images.githubusercontent.com/60723932/173186451-30fcf60f-faab-4e5b-8206-233538751ca8.png)

Reply:
> in our previous project : import { Flowers } from "./Flowers";
> 
> it worked for us also we had in the end: export default App;

Can you check pls how the Flowers is exported in the previous project?
did I use `default` keyword also or just `export` ?

Reply:
I have scrutinized this point before asking you and I found 
we used :
export default App;

but I did not open Flowers.js file 
today when I had a look no we did not use default